padl/transforms.py

```diff
@@ -37,6 +40,16 @@
     make_green, create_arrow, format_argument, visible_len
 
 
+class _Notset:
+    # pylint: disable=too-few-public-methods
+    def __bool__(self):
+        return False
+
+
+_notset = _Notset()
```

Comment:
I believe `_notset` was removed by @wuhu in an earlier PR. I don't think this is needed anymore, right @wuhu ?

Reply:
yeah, makes sense, I might got confused when solving merging conflicts